src/_scss/pages/search/searchSidebar.scss

```diff
@@ -86,6 +86,11 @@
                         height: rem(15);
                         width: rem(15);
                     }
+                    .tooltip-spacer{
+                        @media (max-width: $medium-screen) {
+                            left: rem(-13) !important;
```

Comment:
Just wondering the purpose for the override here and wondering if this can be refactored so you don't have to use "!important"

Reply:
If I don't have the override loses where the edge of the pages is and goes off page. IN doing so the words go back and forth between lines and its really glitchy looking.  Ill send on slack